framework/source/class/qx/ui/core/Widget.js

```diff
@@ -3687,6 +3687,19 @@ qx.Class.define("qx.ui.core.Widget",
     },
 
 
+    /**
+     * Return the ID of the current widget if it is a created as a 
```

Comment:
yes. But the name is called id in e.g. _createChildControlImpl https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/ui/core/Widget.js#L3629
So I thought it's the right naming